web/packages/design/src/Alert/Alert.jsx

```diff
@@ -45,6 +46,14 @@ const kind = props => {
         background: theme.colors.success,
         color: theme.colors.text.primaryInverse,
       };
+    case 'outline-info':
+      return {
+        background: fade(theme.colors.link, 0.1),
+        border: `${theme.radii[1]}px solid ${theme.colors.link}`,
+        borderRadius: `${theme.radii[3]}px`,
```

Comment:
Having a different border radius here is on purpose?

Reply:
yes, the latest design called for it: https://www.figma.com/file/uLevdNsEnIvvLDSZ9sQqXM/Discover-Access?type=design&node-id=1963-9188&mode=design&t=9oBHExTE75bk4kJA-0